module/applications/activity/summon-usage-dialog.mjs

```diff
@@ -0,0 +1,113 @@
+        name: "create.summons",
+        value: this.config.create?.summons
+      });
+
+      if ( this.config.create?.summons ) {
+        const rollData = this.activity.getRollData();
+        if ( profiles.length > 1 ) {
+          let options = profiles.map(profile => ({
+            value: profile._id, label: this.getProfileLabel(profile, rollData)
+          }));
+          if ( options.every(o => o.label.startsWith("1 × ")) ) {
+            options = options.map(({ value, label }) => ({ value, label: label.replace("1 × ", "") }));
+          }
+          context.summonsFields.push({
+            field: new StringField({ label: game.i18n.localize("DND5E.SUMMON.Profile.Label") }),
+            name: "summons.profile",
+            value: this.config.summons?.profile,
+            options
+          });
+        } else context.summonsProfile = profiles[0]._id;
+
+        if ( this.activity.creatureSizes.size > 1 ) context.summonsFields.push({
+          field: new StringField({ label: game.i18n.localize("DND5E.Size") }),
+          name: "summons.creatureSize",
+          value: this.config.summons?.creatureSize,
+          options: Array.from(this.activity.creatureSizes)
+            .map(value => ({ value, label: CONFIG.DND5E.actorSizes[value]?.label }))
+            .filter(k => k)
+        });
+
+        if ( this.activity.creatureTypes.size > 1 ) context.summonsFields.push({
+          field: new StringField({ label: game.i18n.localize("DND5E.CreatureType") }),
+          name: "summons.creatureType",
+          value: this.config.summons?.creatureType,
+          options: Array.from(this.activity.creatureTypes)
+            .map(k => ({ value, label: CONFIG.DND5E.creatureTypes[value]?.label }))
+            .filter(k => k)
+        });
+      }
+    }
+
+    return context;
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Determine the label for a profile in the ability use dialog.
+   * @param {SummonsProfile} profile  Profile for which to generate the label.
+   * @param {object} rollData         Roll data used to prepare the count.
+   * @returns {string}
+   */
+  getProfileLabel(profile, rollData) {
+    let label;
+    if ( profile.name ) label = profile.name;
+    else {
+      switch ( this.activity.summon.mode ) {
+        case "cr":
+          const cr = simplifyBonus(profile.cr, rollData);
+          label = game.i18n.format("DND5E.SUMMON.Profile.ChallengeRatingLabel", { cr: formatCR(cr) });
+          break;
+        default:
+          const doc = fromUuidSync(profile.uuid);
+          if ( doc ) label = doc.name;
+          break;
+      }
+    }
+    label ??= "—";
+
+    let count = simplifyRollFormula(Roll.replaceFormulaData(profile.count ?? "1", rollData));
+    if ( Number.isNumeric(count) ) {
+      count = parseInt(count);
+      label = `${count} × ${label}`;
+    } else if ( count ) label = `${count} × ${label}`;
```

Comment:
Looks like, since we have no method of enforcing that a formula contains an integer. I added an extra check in `getPlacement` to avoid any errors if the formula results in a non-integer value.